pyproject.toml

```diff
@@ -85,9 +85,10 @@ dev = [
 torchvision = [
     "torchvision",
 ] #Leaving torchvisions dependency unconstrained so uv can just install something that should work for the torch we have. TV's on PyT makes it hard to put version constrains in
-quantization = ["nvidia-modelopt[deploy,hf,torch]~=0.17.0"]
+quantization = ["nvidia-modelopt[deploy,hf,torch]>=0.17.0"]
 monitoring-tools = ["rich>=13.7.1"]
 jupyter = ["rich[jupyter]>=13.7.1"]
+distributed = ["tensorrt-llm>=0.16.0"]
 
```

Comment:
How do we ensure this goes through when we have a torchTRT container having python version different than 3.10. Won't this cause an issue here?

Reply:
Its an extra so it wont effect install